videojs/http-streaming
# fix: pause/abort loaders before an exclude, prevent bad appends

## Description
1. Pause loading needs to call abort as well, or else we seem to still be able to continue requesting segments after an error. All of the segments will fail to append, but we will waste a lot of bandwidth.
2. We call pauseLoading after an exclusion to prevent the excluded playlist from appending a bad segment. 


## Why didn't this come up sooner
1. When appending same-codec content appends don't really error, different codec content though does.

## Testing
1. Go to the index page in main change renditions to the hevc renditions as fast as possible and notice an append error.
2. Try the same on this branch
